libs/interpret/src/lib/MLIDashboard/Controls/GlobalExplanationTab/GlobalExplanationTab.styles.ts

```diff
@@ -77,6 +77,7 @@ export const globalTabStyles: () => IProcessedStyleSet<IGlobalTabStyles> =
       },
       legendAndSort: {
         height: "100%",
+        paddingLeft: "25px",
```

Comment:
Did you check the individual explanation dashboard in test app or notebooks?  I'm worried this might be pushing it out too far, where it can get cut off when shown in-notebook.

Reply:
I noticed the new individual datapoints chart in dataset explorer is actually getting cut off when rendered in-notebook.  When viewing inside a jupyter notebook cell we really don't have an much horizontal space to work with.

Reply:
looking at the test app (from the link https://responsibleai.blob.core.windows.net/pullrequest/microsoft/responsible-ai-toolbox/tongy/alignmentIssues/dashboard/index.html) some of the alignments look really off, but I'm not sure if it is due to this PR or it was that way before:

![image](https://user-images.githubusercontent.com/24683184/163870346-f6b2621c-0bfa-49c6-945a-3a1788e7a53b.png)

Reply:
![image](https://user-images.githubusercontent.com/91754176/163873127-9a47c35c-6b10-4247-bc9d-71b610a60455.png)
Tested in the notebook, the View dependence plot looks good. Great comment on the new test scenario that I never thought about